<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server CIVETWEB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Basic control:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartServer">
        <property name="text">
         <string>Server start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopServer">
        <property name="text">
         <string>Server stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRestartServer">
        <property name="text">
         <string>Server restart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestDbConnect">
        <property name="text">
         <string>DB connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDBdisconnect">
        <property name="text">
         <string>DB disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestHTTP">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestHTTPS">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenBrowserHTTP">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenBrowserHTTPS">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadSSLCert">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerateTestCert">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSSLStatus">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Port configuration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPortHTTP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>HTTP not  used</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPortHTTPS">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>8443</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Functionality testing:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Get Hash from Pass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestFileServe">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestAPI">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>reserve</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Information and logs:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblServerStatus">
        <property name="text">
         <string>server status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDBstatus">
        <property name="text">
         <string>DB status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearLogs">
        <property name="text">
         <string>Clear logs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
